README.md

```diff
@@ -1,14 +1,14 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
+- Recursion  - Recursion is when a method calls itself (opposite of iterative).
```

Comment:
I wouldn't say "opposite" of iterative. Maybe "opposed" to iterative. But opposite seems weird to me.